### ISSUE TITLE
Improve accessibility of accordion component

### ISSUE THREAD

**Issue by paulcpederson** [Esri/calcite-web#978]
JAWS does not announce the expanded/collapsed state of an accordion section title. 

![379cd542-9c83-11e8-9de4-7e78ea9c112e](https://user-images.githubusercontent.com/1031758/44751567-5b34ba80-aacd-11e8-8a99-d2a33d709086.png)

As per the [W3C Example](http://w3c.github.io/aria-practices/examples/accordion/accordion.html), we need to move the `aria-expanded` from the `<section>` element to the `<h4>` element. The `<h4>` element also needs to have `aria-controls` set to the `id` of the `<ul>` of the filters. These `id`s should be auto-generated unique id's.

Current code:

```html
<section class="accordion-section is-active" aria-expanded="true">
  <h4 class="accordion-title" role="tab" tabindex="0"> ... </h4>
  <div class="accordion-content filter">...</div>
``` 

Correct:

```html
<section class="accordion-section is-active" >
  <h4 class="accordion-title" role="tab" tabindex="0" aria-expanded="true" aria-controls="panelid"> ... </h4>
  <div class="accordion-content filter" id="panelid">...</div>
```

**Comment by paulcpederson** [comment]
addressed in v1.1.0
